### ISSUE TITLE
macOS App Store always updating the same apps

### ISSUE THREAD

**Issue by REBELinBLUE** [r-darwish/topgrade#700]
**What did you expect to happen?**
Install macOS App Store updates once

**What actually happened?**
Whenever I run the macOS App Store update step it always says the same 3 apps are being updated, if I look at the App Store itself it doesn't say there are any updates

<img width="701" alt="Screenshot 2021-04-26 at 21 18 07" src="https://user-images.githubusercontent.com/2143908/116145468-4a510280-a6d5-11eb-9a99-45c56a00ca9f.png">

<img width="708" alt="Screenshot 2021-04-26 at 21 19 42" src="https://user-images.githubusercontent.com/2143908/116145476-4c1ac600-a6d5-11eb-92ea-8b668ae7bb8b.png">

**Additional details:**
- How did you install Topgrade?  Via home-brew

Run `topgrade --dry-run` to see which commands Topgrade is running. If the command seems wrong and you know why pleaes tell us so.
If the command seems fine try to run it yourself and tell us if you got a different result from Topgrade.

<details><pre>
❯ topgrade -v
 2021-04-26T20:21:17.878Z DEBUG topgrade::config > Configuration at /Users/stephen/.config/topgrade.toml
 2021-04-26T20:21:17.879Z DEBUG topgrade::config > Loaded configuration: ConfigFile { pre_commands: None, post_commands: None, commands: None, git_repos: None, predefined_git_repos: None, disable: Some([Composer, Node, GitRepos, Vscode]), ignore_failures: None, remote_topgrades: None, remote_topgrade_path: None, ssh_arguments: None, git_arguments: None, tmux_arguments: None, set_title: Some(true), assume_yes: Some(true), yay_arguments: None, no_retry: None, run_in_tmux: None, cleanup: Some(true), notify_each_step: Some(false), accept_all_windows_updates: None, bashit_branch: None, only: None, composer: None, brew: None, linux: None, git: None, windows: None, vagrant: None }
 2021-04-26T20:21:17.880Z DEBUG topgrade         > Version: 6.8.0
 2021-04-26T20:21:17.880Z DEBUG topgrade         > OS: x86_64-apple-darwin
 2021-04-26T20:21:17.880Z DEBUG topgrade         > Args { inner: ["topgrade", "-v"] }
 2021-04-26T20:21:17.880Z DEBUG topgrade         > Binary path: Ok("/usr/local/bin/topgrade")
 2021-04-26T20:21:17.880Z DEBUG topgrade         > Self Update: false
 2021-04-26T20:21:17.880Z DEBUG topgrade::utils  > Detected "/usr/local/bin/git" as "git"
 2021-04-26T20:21:17.880Z DEBUG topgrade::utils  > Cannot find "doas"
 2021-04-26T20:21:17.881Z DEBUG topgrade::utils  > Detected "/usr/bin/sudo" as "sudo"
 2021-04-26T20:21:17.881Z DEBUG topgrade::utils  > Cannot find "pwsh"
 2021-04-26T20:21:17.881Z DEBUG topgrade::utils  > Cannot find "powershell"
 2021-04-26T20:21:17.881Z DEBUG topgrade::runner > Step "Brew (ARM)"
 2021-04-26T20:21:17.881Z DEBUG topgrade::runner > Step "Brew (Intel)"
 2021-04-26T20:21:17.882Z DEBUG topgrade::utils  > Detected "/usr/local/bin/brew" as "/usr/local/bin/brew"

―― 21:21:17 - Brew ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-04-26T20:21:17.882Z DEBUG topgrade::executor > Running "/usr/local/bin/brew" "update"
Already up-to-date.
 2021-04-26T20:21:19.400Z DEBUG topgrade::executor > Running "/usr/local/bin/brew" "upgrade" "--ignore-pinned" "--formula"
 2021-04-26T20:21:21.484Z DEBUG topgrade::executor > Running "/usr/local/bin/brew" "cleanup"
 2021-04-26T20:21:38.664Z DEBUG topgrade::runner   > Step "Brew Cask (ARM)"
 2021-04-26T20:21:38.664Z DEBUG topgrade::runner   > Step "Brew Cask (Intel)"
 2021-04-26T20:21:38.664Z DEBUG topgrade::utils    > Detected "/usr/local/bin/brew" as "/usr/local/bin/brew"

―― 21:21:38 - Brew - Cask ――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-04-26T20:21:39.911Z DEBUG topgrade::executor > Running "/usr/local/bin/brew" "upgrade" "--cask"
 2021-04-26T20:21:41.397Z DEBUG topgrade::executor > Running "/usr/local/bin/brew" "cleanup"
 2021-04-26T20:21:58.506Z DEBUG topgrade::runner   > Step "MacPorts"
 2021-04-26T20:21:58.513Z DEBUG topgrade::runner   > Step "Microsoft AutoUpdate"
 2021-04-26T20:21:58.513Z DEBUG topgrade::runner   > Step "yadm"
 2021-04-26T20:21:58.513Z DEBUG topgrade::runner   > Step "nix"
 2021-04-26T20:21:58.514Z DEBUG topgrade::runner   > Step "home-manager"
 2021-04-26T20:21:58.514Z DEBUG topgrade::runner   > Step "asdf"
 2021-04-26T20:21:58.515Z DEBUG topgrade::utils    > Path "/Users/stephen/.config/emacs" doesn't exist
 2021-04-26T20:21:58.515Z DEBUG topgrade::utils    > Path "/Users/stephen/.config/emacs" doesn't exist
 2021-04-26T20:21:58.515Z DEBUG topgrade::utils    > Path "/Users/stephen/.emacs.d" doesn't exist
 2021-04-26T20:21:58.515Z DEBUG topgrade::steps::git > /Users/stephen/.doom.d does not exists
 2021-04-26T20:21:58.515Z DEBUG topgrade::steps::git > /Users/stephen/.vim does not exists
 2021-04-26T20:21:58.515Z DEBUG topgrade::steps::git > /Users/stephen/.config/nvim does not exists
 2021-04-26T20:21:58.515Z DEBUG topgrade::steps::git > /Users/stephen/.zshrc does not exists
 2021-04-26T20:21:58.515Z DEBUG topgrade::steps::git > /Users/stephen/.tmux does not exists
 2021-04-26T20:21:58.515Z DEBUG topgrade::steps::git > Checking if /Users/stephen/.dotfiles/files/shell/fish is a git repository
 2021-04-26T20:21:58.522Z TRACE topgrade::executor   > Output of "/usr/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/Users/stephen/.dotfiles\n", stderr: "" }
 2021-04-26T20:21:58.523Z DEBUG topgrade::steps::git > /Users/stephen/Library/Application Support/openbox does not exists
 2021-04-26T20:21:58.523Z DEBUG topgrade::steps::git > /Users/stephen/Library/Application Support/bspwm does not exists
 2021-04-26T20:21:58.523Z DEBUG topgrade::steps::git > /Users/stephen/Library/Application Support/i3 does not exists
 2021-04-26T20:21:58.524Z DEBUG topgrade::steps::git > /Users/stephen/Library/Application Support/sway does not exists
 2021-04-26T20:21:58.524Z DEBUG topgrade::runner     > Step "zr"
 2021-04-26T20:21:58.524Z DEBUG topgrade::utils      > Detected "/usr/local/bin/zsh" as "zsh"
 2021-04-26T20:21:58.525Z DEBUG topgrade::runner     > Step "antibody"
 2021-04-26T20:21:58.525Z DEBUG topgrade::utils      > Detected "/usr/local/bin/zsh" as "zsh"
 2021-04-26T20:21:58.525Z DEBUG topgrade::runner     > Step "antigen"
 2021-04-26T20:21:58.525Z DEBUG topgrade::utils      > Detected "/usr/local/bin/zsh" as "zsh"
 2021-04-26T20:21:58.525Z DEBUG topgrade::runner     > Step "zplug"
 2021-04-26T20:21:58.525Z DEBUG topgrade::utils      > Detected "/usr/local/bin/zsh" as "zsh"
 2021-04-26T20:21:58.525Z DEBUG topgrade::runner     > Step "zinit"
 2021-04-26T20:21:58.526Z DEBUG topgrade::utils      > Detected "/usr/local/bin/zsh" as "zsh"
 2021-04-26T20:21:58.526Z DEBUG topgrade::runner     > Step "zim"
 2021-04-26T20:21:58.526Z DEBUG topgrade::utils      > Detected "/usr/local/bin/zsh" as "zsh"
 2021-04-26T20:21:58.543Z TRACE topgrade::executor   > Output of "zsh" "-c" "[[ -n ${ZIM_HOME} ]] && print -n ${ZIM_HOME}": Output { status: ExitStatus(ExitStatus(256)), stdout: "", stderr: "" }
 2021-04-26T20:21:58.543Z DEBUG topgrade::runner     > Step "oh-my-zsh"
 2021-04-26T20:21:58.543Z DEBUG topgrade::utils      > Detected "/usr/local/bin/zsh" as "zsh"
 2021-04-26T20:21:58.543Z DEBUG topgrade::runner     > Step "fisher"
 2021-04-26T20:21:58.543Z DEBUG topgrade::utils      > Detected "/usr/local/bin/fish" as "fish"
 2021-04-26T20:21:58.543Z DEBUG topgrade::utils      > Path "/Users/stephen/.config/fish/functions/fisher.fish" exists

―― 21:21:58 - Fisher ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-04-26T20:21:58.543Z DEBUG topgrade::executor   > Running "/usr/local/bin/fish" "-c" "fisher update"
fisher update version 4.3.0
Fetching https://codeload.github.com/edc/bass/tar.gz/HEAD
Fetching https://codeload.github.com/FabioAntunes/fish-nvm/tar.gz/HEAD
Fetching https://codeload.github.com/jorgebucaran/fisher/tar.gz/HEAD
Installing edc/bass
           /Users/stephen/.config/fish/functions/__bass.py
           /Users/stephen/.config/fish/functions/bass.fish
Installing FabioAntunes/fish-nvm
           /Users/stephen/.config/fish/functions/__nvm_run.fish
           /Users/stephen/.config/fish/functions/node.fish
           /Users/stephen/.config/fish/functions/npm.fish
           /Users/stephen/.config/fish/functions/npx.fish
           /Users/stephen/.config/fish/functions/nvm.fish
           /Users/stephen/.config/fish/functions/nvm_alias_command.fish
           /Users/stephen/.config/fish/functions/nvm_alias_function.fish
           /Users/stephen/.config/fish/functions/yarn.fish
           /Users/stephen/.config/fish/completions/nvm.fish
Installing jorgebucaran/fisher
           /Users/stephen/.config/fish/functions/fisher.fish
           /Users/stephen/.config/fish/completions/fisher.fish
Updated 3 plugin/s
 2021-04-26T20:22:00.411Z DEBUG topgrade::runner     > Step "bash-it"
 2021-04-26T20:22:00.411Z DEBUG topgrade::runner     > Step "oh-my-fish"
 2021-04-26T20:22:00.411Z DEBUG topgrade::utils      > Detected "/usr/local/bin/fish" as "fish"
 2021-04-26T20:22:00.411Z DEBUG topgrade::runner     > Step "tmux"
 2021-04-26T20:22:00.411Z DEBUG topgrade::runner     > Step "TLDR"
 2021-04-26T20:22:00.412Z DEBUG topgrade::utils      > Detected "/usr/local/bin/tldr" as "tldr"

―― 21:22:00 - TLDR ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-04-26T20:22:00.412Z DEBUG topgrade::executor   > Running "/usr/local/bin/tldr" "--update"
 2021-04-26T20:22:02.297Z DEBUG topgrade::runner     > Step "pearl"
 2021-04-26T20:22:02.297Z DEBUG topgrade::runner     > Step "SDKMAN!"
 2021-04-26T20:22:02.297Z DEBUG topgrade::utils      > Detected "/usr/local/bin/bash" as "bash"
 2021-04-26T20:22:02.297Z DEBUG topgrade::runner     > Step "apm"
 2021-04-26T20:22:02.298Z DEBUG topgrade::runner     > Step "vscodium"
 2021-04-26T20:22:02.298Z DEBUG topgrade::runner     > Step "fossil"
 2021-04-26T20:22:02.298Z DEBUG topgrade::runner     > Step "rustup"
 2021-04-26T20:22:02.299Z DEBUG topgrade::runner     > Step ".NET"
 2021-04-26T20:22:02.299Z DEBUG topgrade::runner     > Step "choosenim"
 2021-04-26T20:22:02.299Z DEBUG topgrade::runner     > Step "cargo"
 2021-04-26T20:22:02.299Z DEBUG topgrade::runner     > Step "Flutter"
 2021-04-26T20:22:02.300Z DEBUG topgrade::runner     > Step "Emacs"
 2021-04-26T20:22:02.300Z DEBUG topgrade::runner     > Step "opam"
 2021-04-26T20:22:02.300Z DEBUG topgrade::runner     > Step "vcpkg"
 2021-04-26T20:22:02.301Z DEBUG topgrade::runner     > Step "pipx"
 2021-04-26T20:22:02.301Z DEBUG topgrade::runner     > Step "pip3"
 2021-04-26T20:22:02.301Z DEBUG topgrade::utils      > Detected "/usr/local/bin/pip3" as "pip3"

―― 21:22:02 - pip3 ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-04-26T20:22:02.301Z DEBUG topgrade::executor   > Running "/usr/local/bin/pip3" "install" "--upgrade" "--user" "pip"
Requirement already satisfied: pip in ./Library/Python/3.9/lib/python/site-packages (21.1)
WARNING: Value for scheme.platlib does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
distutils: /Users/stephen/Library/Python/3.9/lib/python/site-packages
sysconfig: /Users/stephen/Library/Python/3.9/lib/python3.9/site-packages
WARNING: Value for scheme.purelib does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
distutils: /Users/stephen/Library/Python/3.9/lib/python/site-packages
sysconfig: /Users/stephen/Library/Python/3.9/lib/python3.9/site-packages
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
distutils: /Users/stephen/Library/Python/3.9/include/python3.9/UNKNOWN
sysconfig: /Users/stephen/Library/Python/3.9/include/python3.9
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
 2021-04-26T20:22:03.343Z DEBUG topgrade::runner     > Step "stack"
 2021-04-26T20:22:03.344Z DEBUG topgrade::runner     > Step "tlmgr"
 2021-04-26T20:22:03.344Z DEBUG topgrade::runner     > Step "myrepos"
 2021-04-26T20:22:03.345Z DEBUG topgrade::runner     > Step "jetpack"
 2021-04-26T20:22:03.345Z DEBUG topgrade::runner     > Step "vim"
 2021-04-26T20:22:03.346Z DEBUG topgrade::utils      > Detected "/usr/local/opt/vim/bin/vim" as "vim"
 2021-04-26T20:22:03.367Z TRACE topgrade::executor   > Output of "/usr/local/opt/vim/bin/vim" "--version": Output { status: ExitStatus(ExitStatus(0)), stdout: "VIM - Vi IMproved 8.2 (2019 Dec 12, compiled Apr 22 2021 19:39:30)\nmacOS version - x86_64\nIncluded patches: 1-2800\nCompiled by Homebrew\nHuge version without GUI.  Features included (+) or not (-):\n+acl               -farsi             +mouse_sgr         +tag_binary\n+arabic            +file_in_path      -mouse_sysmouse    -tag_old_static\n+autocmd           +find_in_path      +mouse_urxvt       -tag_any_white\n+autochdir         +float             +mouse_xterm       -tcl\n-autoservername    +folding           +multi_byte        +termguicolors\n-balloon_eval      -footer            +multi_lang        +terminal\n+balloon_eval_term +fork()            -mzscheme          +terminfo\n-browse            +gettext           +netbeans_intg     +termresponse\n++builtin_terms    -hangul_input      +num64             +textobjects\n+byte_offset       +iconv             +packages          +textprop\n+channel           +insert_expand     +path_extra        +timers\n+cindent           +ipv6              +perl              +title\n-clientserver      +job               +persistent_undo   -toolbar\n+clipboard         +jumplist          +popupwin          +user_commands\n+cmdline_compl     +keymap            +postscript        +vartabs\n+cmdline_hist      +lambda            +printer           +vertsplit\n+cmdline_info      +langmap           +profile           +virtualedit\n+comments          +libcall           -python            +visual\n+conceal           +linebreak         +python3           +visualextra\n+cryptv            +lispindent        +quickfix          +viminfo\n+cscope            +listcmds          +reltime           +vreplace\n+cursorbind        +localmap          +rightleft         +wildignore\n+cursorshape       +lua               +ruby              +wildmenu\n+dialog_con        +menu              +scrollbind        +windows\n+diff              +mksession         +signs             +writebackup\n+digraphs          +modify_fname      +smartindent       -X11\n-dnd               +mouse             -sound             -xfontset\n-ebcdic            -mouseshape        +spell             -xim\n+emacs_tags        +mouse_dec         +startuptime       -xpm\n+eval              -mouse_gpm         +statusline        -xsmp\n+ex_extra          -mouse_jsbterm     -sun_workshop      -xterm_clipboard\n+extra_search      +mouse_netterm     +syntax            -xterm_save\n   system vimrc file: \"$VIM/vimrc\"\n     user vimrc file: \"$HOME/.vimrc\"\n 2nd user vimrc file: \"~/.vim/vimrc\"\n      user exrc file: \"$HOME/.exrc\"\n       defaults file: \"$VIMRUNTIME/defaults.vim\"\n  fall-back for $VIM: \"/usr/local/share/vim\"\nCompilation: clang -c -I. -Iproto -DHAVE_CONFIG_H -DMACOS_X -DMACOS_X_DARWIN -g -O2 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 \nLinking: clang -L. -fstack-protector-strong -L/usr/local/lib -L/usr/local/opt/libyaml/lib -L/usr/local/opt/openssl@1.1/lib -L/usr/local/opt/readline/lib -L/usr/local/lib -o vim -lm -lncurses -liconv -lintl -framework AppKit -L/usr/local/opt/lua/lib -llua5.4 -mmacosx-version-min=11.2 -fstack-protector-strong -L/usr/local/lib -L/usr/local/Cellar/perl/5.32.1_1/lib/perl5/5.32.1/darwin-thread-multi-2level/CORE -lperl -L/usr/local/opt/python@3.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/config-3.9-darwin -lpython3.9 -framework CoreFoundation -lruby.3.0 -L/usr/local/Cellar/ruby/3.0.1/lib \n", stderr: "" }
 2021-04-26T20:22:03.368Z DEBUG topgrade::runner     > Step "Neovim"
 2021-04-26T20:22:03.368Z DEBUG topgrade::runner     > Step "voom"
 2021-04-26T20:22:03.368Z DEBUG topgrade::runner     > Step "deno"
 2021-04-26T20:22:03.369Z DEBUG topgrade::runner     > Step "krew"
 2021-04-26T20:22:03.369Z DEBUG topgrade::utils      > Detected "/Users/stephen/.krew/bin/kubectl-krew" as "kubectl-krew"

―― 21:22:03 - Krew ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-04-26T20:22:03.369Z DEBUG topgrade::executor   > Running "/Users/stephen/.krew/bin/kubectl-krew" "upgrade"
Updated the local copy of plugin index.
Upgrading plugin: access-matrix
Skipping plugin access-matrix, it is already on the newest version
Upgrading plugin: get-all
Skipping plugin get-all, it is already on the newest version
Upgrading plugin: images
Skipping plugin images, it is already on the newest version
Upgrading plugin: krew
Skipping plugin krew, it is already on the newest version
Upgrading plugin: kubesec-scan
Skipping plugin kubesec-scan, it is already on the newest version
Upgrading plugin: neat
Skipping plugin neat, it is already on the newest version
Upgrading plugin: outdated
Skipping plugin outdated, it is already on the newest version
Upgrading plugin: pod-dive
Skipping plugin pod-dive, it is already on the newest version
Upgrading plugin: rbac-lookup
Skipping plugin rbac-lookup, it is already on the newest version
Upgrading plugin: resource-capacity
Skipping plugin resource-capacity, it is already on the newest version
Upgrading plugin: tree
Skipping plugin tree, it is already on the newest version
Upgrading plugin: who-can
Skipping plugin who-can, it is already on the newest version
 2021-04-26T20:22:04.062Z DEBUG topgrade::runner     > Step "gem"
 2021-04-26T20:22:04.063Z DEBUG topgrade::utils      > Detected "/usr/bin/gem" as "gem"
 2021-04-26T20:22:04.063Z DEBUG topgrade::utils      > Path "/Users/stephen/.gem" exists

―― 21:22:04 - RubyGems ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-04-26T20:22:04.063Z DEBUG topgrade::steps::generic > Detected rbenv. Avoiding --user-install
 2021-04-26T20:22:04.063Z DEBUG topgrade::executor       > Running "/usr/bin/gem" "update" "--user-install"
Updating installed gems
Nothing to update
 2021-04-26T20:22:06.942Z DEBUG topgrade::runner         > Step "sheldon"
 2021-04-26T20:22:06.947Z DEBUG topgrade::runner         > Step "rtcl"
 2021-04-26T20:22:06.948Z DEBUG topgrade::runner         > Step "gcloud"
 2021-04-26T20:22:06.948Z DEBUG topgrade::runner         > Step "micro"
 2021-04-26T20:22:06.949Z DEBUG topgrade::runner         > Step "raco"
 2021-04-26T20:22:06.949Z DEBUG topgrade::runner         > Step "App Store"
 2021-04-26T20:22:06.949Z DEBUG topgrade::utils          > Detected "/usr/local/bin/mas" as "mas"

―― 21:22:06 - macOS App Store ――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-04-26T20:22:06.949Z DEBUG topgrade::executor       > Running "/usr/local/bin/mas" "upgrade"
Upgrading 3 outdated applications:
Messenger (97.11.116), HomeCam (2.1.4), Instapaper (8.0.1)
==> Downloading Messenger
==> Installed Messenger
==> Downloading HomeCam for HomeKit
==> Installed HomeCam for HomeKit
==> Downloading Instapaper
==> Installed Instapaper
 2021-04-26T20:22:28.788Z DEBUG topgrade::runner         > Step "System upgrade"

―― 21:22:28 - macOS system update ――――――――――――――――――――――――――――――――――――――――――――――
 2021-04-26T20:22:28.789Z DEBUG topgrade::executor       > Running "softwareupdate" "--install" "--all"
Software Update Tool

Finding available software
No updates are available.

―― 21:22:32 - Summary ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
Brew (ARM): SKIPPED: Cannot find "/opt/homebrew/bin/brew" in PATH
Brew (Intel): OK
Brew Cask (ARM): SKIPPED: Cannot find "/opt/homebrew/bin/brew" in PATH
Brew Cask (Intel): OK
MacPorts: SKIPPED: Cannot find "port" in PATH
Microsoft AutoUpdate: SKIPPED: Path "/Library/Application Support/Microsoft/MAU2.0/Microsoft AutoUpdate.app/Contents/MacOS/msupdate" doesn't exist
yadm: SKIPPED: Cannot find "yadm" in PATH
nix: SKIPPED: Cannot find "nix" in PATH
home-manager: SKIPPED: Cannot find "home-manager" in PATH
asdf: SKIPPED: Cannot find "asdf" in PATH
zr: SKIPPED: Cannot find "zr" in PATH
antibody: SKIPPED: Cannot find "antibody" in PATH
antigen: SKIPPED: Path "/Users/stephen/.zshrc" doesn't exist
zplug: SKIPPED: Path "/Users/stephen/.zshrc" doesn't exist
zinit: SKIPPED: Path "/Users/stephen/.zshrc" doesn't exist
zim: SKIPPED: Path "/Users/stephen/.zim" doesn't exist
oh-my-zsh: SKIPPED: Path "/Users/stephen/.oh-my-zsh" doesn't exist
fisher: OK
bash-it: SKIPPED: Path "/Users/stephen/.bash_it" doesn't exist
oh-my-fish: SKIPPED: Path "/Users/stephen/.local/share/omf/pkg/omf/functions/omf.fish" doesn't exist
tmux: SKIPPED: Path "/Users/stephen/.tmux/plugins/tpm/bin/update_plugins" doesn't exist
TLDR: OK
pearl: SKIPPED: Cannot find "pearl" in PATH
SDKMAN!: SKIPPED: Path "/Users/stephen/.sdkman/bin/sdkman-init.sh" doesn't exist
apm: SKIPPED: Cannot find "apm" in PATH
vscodium: SKIPPED: Cannot find "codium" in PATH
fossil: SKIPPED: Cannot find "fossil" in PATH
rustup: SKIPPED: Cannot find "rustup" in PATH
.NET: SKIPPED: Cannot find "dotnet" in PATH
choosenim: SKIPPED: Cannot find "choosenim" in PATH
cargo: SKIPPED: Cannot find "cargo" in PATH
Flutter: SKIPPED: Cannot find "flutter" in PATH
Emacs: SKIPPED: Cannot find "emacs" in PATH
opam: SKIPPED: Cannot find "opam" in PATH
vcpkg: SKIPPED: Cannot find "vcpkg" in PATH
pipx: SKIPPED: Cannot find "pipx" in PATH
pip3: OK
stack: SKIPPED: Cannot find "stack" in PATH
tlmgr: SKIPPED: Cannot find "tlmgr" in PATH
myrepos: SKIPPED: Cannot find "mr" in PATH
jetpack: SKIPPED: Cannot find "jetpack" in PATH
vim: SKIPPED: Path "/Users/stephen/.vim/vimrc" doesn't exist
Neovim: SKIPPED: Cannot find "nvim" in PATH
voom: SKIPPED: Cannot find "voom" in PATH
deno: SKIPPED: Cannot find "deno" in PATH
krew: OK
gem: OK
sheldon: SKIPPED: Cannot find "sheldon" in PATH
rtcl: SKIPPED: Cannot find "rupdate" in PATH
gcloud: SKIPPED: Cannot find "gcloud" in PATH
micro: SKIPPED: Cannot find "micro" in PATH
raco: SKIPPED: Cannot find "raco" in PATH
App Store: OK
System upgrade: OK
 2021-04-26T20:22:32.619Z DEBUG topgrade::terminal       > Desktop notification: Topgrade finished sucessfully
</pre></details>

**Comment by REBELinBLUE** [comment]
Just realised, this is something in `mas` not `topgrade` 🤦‍♂️
